docs/source/distributions.rst

```diff
@@ -685,6 +692,10 @@ neural_autoregressive
 ---------------------
 .. autofunction:: pyro.distributions.transforms.neural_autoregressive
 
+ordered
```

Comment:
It looks like docs is failing because you haven't defined a wrapper function `ordered` as many of the other transforms have defined.

Reply:
Haha just noticed that. I did have one defined, but then @fehiepsi suggested I remove it because those helpers are typically reserved for normalizing flows. Forgot to take it out of the docs.